doc/cla/corporate/moduon.md

```diff
@@ -16,3 +16,4 @@ moduonbot moduonbot@moduon.team https://github.com/moduonbot
 Rafael Blasco rblasco@moduon.team https://github.com/rafaelbn
 Andrea Cattalani andrea@moduon.team https://github.com/anddago78 (up to 2024-01-31)
 Emilio Pascual emilio@moduon.team https://github.com/emiliopascual
+David Vidal david.vidal@moduon.team https://github.com/chienandalu
```

Comment:
Please change this e-mail, which is wrong.

```suggestion
David Vidal david@moduon.team https://github.com/chienandalu
```

We'll open the PR for branch 16.0.

Reply:
I let you make the correction for the versions where it’s needed